### PR TITLE
 base_country_state_translatable Module

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

File: base_country_state_translatable/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/9.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/l10n-germany/issues>`_. In case of trouble, please
```
Comment (yajo) [inline base_country_state_translatable/README.rst]: Wrong repo!

File: base_country_state_translatable/models/res_country.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# © initOS GmbH 2017
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class CountryState(models.Model):
+    _inherit = 'res.country.state'
+
+    name = fields.Char(translate=True)
```
Comment (yajo) [inline base_country_state_translatable/models/res_country.py:11]: OK, so this one is a simple addon, but we have to maintain it!

May I ask you please to open a pull request to odoo master and propose the change there? I'm pretty sure they'll merge it, and it's [just 1 line of code on their side.
](https://github.com/odoo/odoo/blob/a7fa3cc436517d001717de6b4402c62698b45224/odoo/addons/base/res/res_country.py#L110-L111)

Once you do it, add a note to the roadmap saying that this addon should be dropped in the version where that PR gets merged.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
Check travis please

File: base_country_state_translatable/models/res_country.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © initOS GmbH 2017
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
```
Comment (pedrobaeza) [inline base_country_state_translatable/models/res_country.py]: `from openerp...`

**Comment by rami-wafaie** [comment]
Reuested Changes are done
+
a PR to Odoo master has been created
https://github.com/odoo/odoo/pull/19481

File: base_country_state_translatable/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © initOS GmbH 2017
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Translate Country States",
+    "version": "9.0.1.0.0",
+    "depends": [],
```
Comment (pedrobaeza) [inline base_country_state_translatable/__openerp__.py]: Should depend on base

**Review by pedrobaeza — APPROVED** [review]

File: base_country_state_translatable/__openerp__.py [hunk 5]
```diff
@@ -5,7 +5,8 @@
 {
     "name": "Translate Country States",
     "version": "9.0.1.0.0",
-    "depends": [],
+    "depends": ['base',
+                ],
```
Comment (yajo) [inline base_country_state_translatable/__openerp__.py]: Woah! that's ugly! Please indent beautifully 😊

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by rami-wafaie** [comment]
Requested changes are done

**Review by yajo — APPROVED** [review]

**Pull request merged**